### PR TITLE
Send email assign trainee

### PR DESCRIPTION
**Pull request by ttuan**

### CODE REVIEW

**Comment by hienbx94** [comment]
Ok

**Comment by trantube94** [comment]
OK

File: app/controllers/admin/assign_trainees_controller.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ def edit
   def update
     if @course.update_attributes user_ids_course_params
       flash[:success] = t "assign_trainees.success"
+      EmailAssignTrainees.perform_async user_ids_course_params[:user_ids], @course.id
```
Comment (dieunb) [inline app/controllers/admin/assign_trainees_controller.rb]: move to call_back

File: app/controllers/admin/user_courses_controller.rb [hunk 2]
```diff
@@ -2,8 +2,10 @@ class Admin::UserCoursesController < ApplicationController
   before_action :require_admin
 
   def destroy
-    if UserCourse.find(params[:id]).destroy
+    user_course = UserCourse.find params[:id]
+    if user_course.destroy
       flash[:success] =  t "user_course.removed_user"    
+      EmailRemoveUserFromCourse.perform_async user_course[:user_id], user_course[:course_id]
```
Comment (dieunb) [inline app/controllers/admin/user_courses_controller.rb]: call_back

File: app/controllers/admin/users_controller.rb [hunk 3]
```diff
@@ -18,7 +18,7 @@ def new
   def create
     @user = User.new user_params
     if @user.save
-      UserMailer.create_account(@user).deliver_now
+      EmailCreatedAccount.perform_async @user.id
```
Comment (dieunb) [inline app/controllers/admin/users_controller.rb]: call_back

File: app/workers/email_assign_trainees.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+class EmailAssignTrainees
+  include Sidekiq::Worker
+
+  def perform user_ids, course_id
+    course = Course.find course_id
+    user_ids.each do |user_id|
```
Comment (dieunb) [inline app/workers/email_assign_trainees.rb]: refactor block code

**Pull request merged**